designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Endpoints.scala

```diff
@@ -173,6 +179,23 @@ class Endpoints(auth: EndpointInput[AuthCredentials], streamProvider: TapirStrea
       .withSecurity(auth)
   }
 
+  val deleteAttachmentEndpoint: SecuredEndpoint[DeleteAttachmentRequest, ScenarioActivityError, Unit, Any] = {
+    baseNuApiEndpoint
+      .summary("Delete scenario attachment service")
```

Comment:
Do we need the word "service" here?

Reply:
removed suffix "service" from all summaries in this API